packages/mdc-menu/simple/foundation.js

```diff
@@ -300,6 +301,14 @@ export default class MDCSimpleMenuFoundation extends MDCFoundation {
   }
 
   handlePossibleSelected_(evt) {
+    const targetIsDisabled =
+      this.adapter_.getAttributeForEventTarget(
+        evt.target, MDCSimpleMenuFoundation.strings.ARIA_DISABLED_ATTR
+      ) === 'true';
+    if (targetIsDisabled) {
+      evt.stopPropagation();
```

Comment:
You know what, I just implemented it that way and I think it's better. That way the user can still listen for events on a disabled item and do whatever custom thing they may want.